index.html

```diff
@@ -0,0 +1,71 @@
+<html>
+
+<head>
+    <title>Nikos Village Websitelol</title>
+</head>
+
+<body>
+    
+    <div class="FadeIn"><h1>Welcome to Drosia!</h1></div>
+  
+    <div class="Fadein"><p>This is the test paragraph for the site. Maybe one day
+        it wont look like it was made in 1999.
+    </p></div>
+
+    <div  class="FadeIn"><button>Test Button</button></div>
+    <script src="Thing.js"></script>
+
+
+    <div class="Fadein"><h2>Cool Things to do in Drosia</h2></div>
+
+    <div class="FadeIn"><ul>
+        <li>Hiking</li>
+        <li>Going out to the village square</li>
+        <li>Climbing mount Erymanthos</li>
+        <li>Insert other stuff here lmao</li>
+      </ul></div>
+
+      <div class="FadeImage"><img src="./View of Drosia.jpg" alt="View of Drosia" class="Center"/></div>
+
+    
+</body>
+
+<style>
```

Comment:
See if you can create a separate `index.css` file and import that into the html ;)